lib/carousel/carousel-control.js

```diff
@@ -31,20 +35,21 @@ const CarouselControl = (props) => {
   const screenReaderClasses = cx(srOnly);
 
   return (
-    <a
+    <button
+      type="button"
       className={anchorClasses}
-      role="button"
       tabIndex="0"
       onClick={(e) => {
         e.preventDefault();
         onClickHandler();
       }}
+      onKeyPress={handleKeyPress}
```

Comment:
Can you create an onKeyUpHandler prop with a default empty function, that the person using this component can provide their own implementation.